version.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
```

Comment:
Just to be clear: this file is something that is maintained by the bot, and *not* something we updated manually?

Reply:
yes correct, this is defined at the next line.